pkg/agent/agent_linux.go

```diff
@@ -105,10 +106,11 @@ func (i *Initializer) prepareOVSBridge() error {
 				return fmt.Errorf("cannot find uplink port %s: err=%w", uplinkPort.Name, err2)
 			}
 			klog.Infof("Found uplink device %s", adapter.Name)
+			i.nodeConfig.UplinkNetConfig.OFPort = uint32(uplinkPort.OFPort)
 			uplinkNetConfig.Name = adapter.Name
 			uplinkNetConfig.Index = adapter.Index
 			uplinkInterface := interfacestore.NewUplinkInterface(uplinkPort.Name)
-			uplinkInterface.OVSPortConfig = &interfacestore.OVSPortConfig{uplinkPort.UUID, config.UplinkOFPort} //nolint: govet
+			uplinkInterface.OVSPortConfig = &interfacestore.OVSPortConfig{uplinkPort.UUID, uplinkPort.OFPort} //nolint: govet
 			i.ifaceStore.AddInterface(uplinkInterface)
 		}
 	} else {
```

Comment:
i.nodeConfig.UplinkNetConfig.OFPort is also needed here. Otherwise the default value 0 will be used.

Reply:
I set it in line 109.

Reply:
That's `if` branch, we also need this in `else` branch.